src/components/MainSidebar/YourDimensions.js

```diff
@@ -0,0 +1,42 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { IconFolder16 } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { MenuItem } from './MenuItem/index.js'
+import {
+    yourDimensionsResource,
+    yourDimensionsFilter,
+} from './YourDimensionsPanel/useYourDimensions.js'
+
+const query = {
+    dimensions: {
+        resource: yourDimensionsResource,
+        params: {
+            page: 1,
+            pageSize: 1,
+            fields: 'id',
+            filter: yourDimensionsFilter,
+        },
+    },
+}
+
+export const YourDimensions = ({ selected, count, onClick }) => {
```

Comment:
Perhaps consider renaming to sth like `YourDimensionsMenuItem`?

Reply:
Done.